source/features/better-diff-selection.tsx

```diff
@@ -0,0 +1,31 @@
+import './better-diff-selection.css';
+
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function disableDiffSelection(event: Event): void {
+	const target = event.target as HTMLElement;
+	target.closest('tbody')!.dataset.rghSelect = target.closest('td:last-child') ? 'right' : 'left';
+}
+
+function restoreDiffSelection(): void {
+	select('[data-rgh-select]')?.removeAttribute('data-rgh-select');
+}
+
+function init(): void {
+	delegate(document.body, '.diff-table tr:not(.js-expandable-line) td:nth-child(even)', 'mousedown', disableDiffSelection);
```

Comment:
Does this work correctly even on .txt files? (the class might be missing if there's no syntax highlighting)
```suggestion
	delegate(document.body, '.blob-code', 'mousedown', disableDiffSelection);
```

Reply:
> Does this work correctly even on .txt files?

Tested on https://github.com/ehedaya/emilstabs.org/pull/200/files, it works. But that selector is much better.